src/shared/views/product.hbs

```diff
@@ -1,2 +1,52 @@
 <h3>{{product_name}}</h3>
 <p>{{product_description}}</p>
+
+<!-- Example pie card mobile markup -->
+<div class="pie-card pie-card--mobile">
+  <div class="fav fav--on">
+    <svg viewBox="0 0 98.325027 85.880819">
```

Comment:
This looks like a lot of the same code that we have in `listing.hbs`. Perhaps we should move this to a partial? Or is there a good reason for keeping two separate copies?

Reply:
The reason is that it's two different card layouts (mobile 320px and small 220px). The markup does differ slightly and I don't see value yet in trying to generalise it. We might consolidate (and e.g. use only 1 type of cards), but I'd rather do it only when the card designs are marked as final by Mustafa.

Reply:
Ah, so there was a difference. If there's definitively nothing shared between both designs as of right now, then yes, I'm ok with keeping them separate.